code/src/ball_c.cpp

```diff
@@ -0,0 +1,39 @@
+#include <ball_c.hpp>
+
+void ball_c::update_ball()
+{
+        x_axis += direction_x;
+        y_axis += direction_y;
+
```

Comment:
Why these empty lines?

Reply:
For demo purposes I'd rather follow my own guidelines

Reply:
ahh, missread comment, removed the empty lines, thanks!